x-pack/plugins/aiops/public/ui_actions/index.ts

```diff
@@ -7,16 +7,29 @@
 
 import type { UiActionsSetup } from '@kbn/ui-actions-plugin/public';
 import { CONTEXT_MENU_TRIGGER } from '@kbn/embeddable-plugin/public';
+import { categorizeFieldTrigger, CATEGORIZE_FIELD_TRIGGER } from '@kbn/ml-ui-actions';
```

Comment:
To keep the page load bundle size of the `aiops` plugin down you might need to do a deep import from the package here otherwise it will add all package code to the bundle size.

Reply:
I'll add this. But it's worth pointing out that the crazy +241.1KB increase was from an [import test](https://github.com/elastic/kibana/pull/171771/commits/9dfd915e17eb3d1c1e45f7d3423c76f2bc35e45a) I tried out just to see if async loading of imports was actually making a difference. It obviously is. I [reverted](https://github.com/elastic/kibana/pull/171771/commits/8f25e609062b83c48f1a4d9ed3b1bf2dd95e1fcb) the test commit